src/components/layouts/DefaultLayout.tsx

```diff
@@ -70,7 +70,7 @@ export function DefaultLayout({ children }: Props) {
       {sidebarLayoutEnabled ? <SidebarNavigation /> : <MarketingNavigation />}
 
       <Content>
-        {/* {sidebarLayoutEnabled && <LargeScreenHeader />} */}
+        {sidebarLayoutEnabled && <LargeScreenHeader />}
```

Comment:
We don't want the top menu back, as it collides with other top menus, such as the one of docs